tests/test_embedder.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import numpy as np
+from PIL import Image
+
+from embedder import EmbedderRuCLIP
+
+def test_encode_text():
```

Comment:
use unittest structure, using `class TestEmbedderRuCLIP(unittest.TestCase)` and then `test_encode_imgs(self)`, and `self.asserEqual` methods

Reply:
ok